ExampleApp/AppDelegate.swift

```diff
@@ -211,6 +216,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         }
     }
+
+    private func handleExampleVerify1FA(verify1FACallbackUrl: URL, host: String) {
+        let VERIFY_1FA = "complete1FA"
+        let YAHOO_HOST = "completeYahoo1FA"
+        if var topViewController = UIApplication.shared.keyWindow?.rootViewController {
+            while let presentedViewController = topViewController.presentedViewController {
+                topViewController = presentedViewController
+            }
+
+            if let navController = topViewController as? UINavigationController,
+                let navTopViewController = navController.topViewController {
+                topViewController = navTopViewController
+            }
```

Comment:
Do we do this calculation somewhere else? Maybe we should create a function for it instead of duplicating